scripts/dev_setup.sh

```diff
@@ -930,6 +957,15 @@ if [[ "$INSTALL_CODEGEN" == "true" ]]; then
   "${PRE_COMMAND[@]}" python3 -m pip install numpy=="${NUMPY_VERSION}"
 fi
 
+if [[ "$INSTALL_API_BUILD_TOOLS" == "true" ]]; then
+  # nodejs and tools
+  install_nodejs
+
+  # python and tools
+  install_python3
+  "${PRE_COMMAND[@]}" python3 -m pip install schemathesis
```

Comment:
this will be used by https://github.com/diem/diem/pull/9659, which is stacked on another PR not landed yet, so pull out dev_setup.sh changes to confirm everything is good for dev_setup script first.